QModManager/HarmonyPatches/QuitToDesktop.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Original mod by RandyKnapp
```

Comment:
Include this link in the comments 
`https://github.com/RandyKnapp/SubnauticaModSystem/tree/master/SubnauticaModSystem/QuitToDesktop`